### DISCUSSION TITLE
Dynamic info from database to _document.js in statically generated Next.js pages

### DISCUSSION THREAD

**Discussion by esteban89** [vercel/next.js#14905] [Help]
I'm trying to define lang property in _document.js but I need to get the site's language from a mySQL database.

Is this possible? Any ideas?

**Comment by jamesmosier** [comment]
This Issue describes some potential work arounds: https://github.com/vercel/next.js/issues/9160#issuecomment-616987683 and https://github.com/vercel/next.js/issues/9160#issuecomment-643867740

**Comment by esteban89** [comment]
> This Issue describes some potential work arounds: [#9160 (comment)](https://github.com/vercel/next.js/issues/9160#issuecomment-616987683) and [#9160 (comment)](https://github.com/vercel/next.js/issues/9160#issuecomment-643867740)

Thanks! Unfortunately that wouldn't work for me since I'm not including the lang in the url (and can't include it).

**Comment by jamesmosier** [comment]
Could you perhaps set it clientside? Where are you currently fetching the data? `getServerSideProps`? `getStaticProps`? Something else?

**Reply by esteban89** [reply]
> Could you perhaps set it clientside? Where are you currently fetching the data? `getServerSideProps`? `getStaticProps`? Something else?

Wouldn't that make it non static?
I'm using getStaticProps to get data from a mySQL db
